### DISCUSSION TITLE
QField Syncing Issues - Layers could not be loaded in QField

### DISCUSSION THREAD

**Discussion by juliette-kool** [opengisch/QField#5498] [Bugs]
I set up a QField project in QGIS by first creating a project and then packaging it for QField. When I sync for the first time it works perfectly and the layers show up in the app as I expect. We have some of the collaborators in the field adding points to layers.

That worked fine last week, but suddenly now all the layers don't show up in the app. the error message is:

> "The following layers could not be loaded, please review those and reconfigure the QGIS project".

![IMG_0071](https://github.com/user-attachments/assets/b0dc166b-1b69-4f65-9478-6ea02f7ec14c)

I am not sure what to do since the field data they acquired is very valuable, and I don't want to lose the data. The files still show up on QField Cloud on the web.

Can someone recommend safe approach to take to make sure that no data is lost?

Machine: Apple M3 Max - MacOS Sonoma Version 14.5 (23F79)
QGIS: 3.34 LTR
QField: 3.3.7

**Comment by nirvn** [comment]
@juliette-kool , you seem to have problematic dataset paths . Any dataset with a path that's _outside_ of the project folder will definitively fail.

E.g., the KBA-Bolaven-hP layer states that the path is "../files/KBBolavenhP.gpkg", which means it's outside of what is being packaged for QField.

**Reply by juliette-kool** [reply]
thank you for the quick response! the strange thing is when I packaged the project initially it was working fine - and even syncing some of the ground-truthing points worked properly, but now when I opened the app this is what I see.

Do you have any recommendations on how to go from here to ensure no data is lost from the points made in the field?

**Comment by Pete-echo** [comment]
I am having exactly the same issue.  I have projects that worked fine last week, but now they won't open in QField.  I have checked all the 'Package for Qfield' settings and they are the same as before, no changes, and I am not doing anything different than what i have done for years of using QGIS and QField.

The only thing I noticed is there has been an update to the QField app in IOS (which I am assuming you are also using given your screen shot).  Maybe this update is causing the problem?  

Windows 11
QGIS 3.38.0
QField 3.3.8

**Reply by Pete-echo** [reply]
I found a solution to this, I uninstalled QGIS and reinstalled a fresh version.  I the used the default user without any of my normal plugins and only installed the Qfield Plugin.   I am now able to create Qfield projects as before and they work without any problems on my iPad.

I think there is an issue with a plugin in my old user profile or its configuration,  I don't know how to find out what this is or even why now it was causing an issue all of a sudden.  But for now I am just happy I can create Qfield projects again!

**Comment by Skipper-is** [comment]
Same issue here. 
QGIS version 3.34.11 LTR, 
QField version 3.4.3

![image](https://github.com/user-attachments/assets/31903715-6572-4b8c-8729-12a966d4038d)
I can verify that those files are in the folder:
![image](https://github.com/user-attachments/assets/277d4d1a-346d-4ce5-a0ce-63941aac48d3)

![image](https://github.com/user-attachments/assets/502d8420-fd3d-4bfb-9e70-77cccf3f86e2)

It is set to relative path in the project, and the locations in the qgs file are:
`<layer-tree-layer source="./UKHAB.gpkg|layername=Pond_HSI"` for example

If I manually edit the qgs file, and remove the ./ it works fine, but that's not a sustainable practice! 

I'm not installing a version of QGIS just to pack projects for QField.. That's silly!
